tests/test_auto.py

```diff
@@ -32,9 +32,12 @@
     PeftModelForSequenceClassification,
     PeftModelForTokenClassification,
 )
+from peft.utils import infer_device
 
 
 class PeftAutoModelTester(unittest.TestCase):
+    dtype = torch.float16 if infer_device() == "mps" else torch.bfloat16
```

Comment:
Would it be possible to check `if torch.cuda.is_bf16_supported()` instead or does that not work on Mac?

Reply:
I wouldn't trust anything that says `cuda` on the tin to work with Macs 😁

Reply:
Maybe that could be held off for later, anyway?